pylint/checkers/refactoring.py

```diff
@@ -1065,10 +1070,14 @@ def visit_compare(self, node):
             error_detected = False
 
             # 0 ?? len()
-            if _is_constant_zero(op_1) and op_2 in ['==', '!=', '<'] and _is_len_call(op_3):
+            if _is_constant_zero(op_1) and op_2 in ['==', '!=', '<', '>='] and _is_len_call(op_3):
                 error_detected = True
             # len() ?? 0
-            elif _is_len_call(op_1) and op_2 in ['==', '!=', '>'] and _is_constant_zero(op_3):
+            elif _is_len_call(op_1) and op_2 in ['==', '!=', '>', '<='] and _is_constant_zero(op_3):
+                error_detected = True
+            elif _has_constant_value(op_1, 1) and op_2 in ['>'] and _is_len_call(op_3):
```

Comment:
You can just check using equality for now instead of checking containment in an one-element list. Also please pass `1` as a keyword argument, so along the lines of `has_contstant_value(op_1, value=1)`